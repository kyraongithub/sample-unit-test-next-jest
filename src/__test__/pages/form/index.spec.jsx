import FormView from '@/components/pages/Form';
import { render } from '@testing-library/react';

describe('Form Page', () => {
  it('should render form page', () => {
    const { container } = render(<FormView />);
    expect(container).toMatchSnapshot();
  });
});
