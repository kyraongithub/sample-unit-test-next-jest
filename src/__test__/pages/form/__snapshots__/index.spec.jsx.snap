// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Page should render form page 1`] = `
<div>
  <div>
    <div>
      <h1>
        Add a user
      </h1>
      <label
        for="Name"
      >
        Name
      </label>
      <br />
      <input
        id="Name"
        placeholder="Enter name"
        type="text"
        value=""
      />
      <br />
      <br />
      <label
        for="Email"
      >
        Email
      </label>
      <br />
      <input
        id="Email"
        placeholder="Enter email"
        type="text"
        value=""
      />
      <br />
      <br />
      <button>
        Submit
      </button>
    </div>
    <hr />
    <table>
      <thead>
        <tr>
          <th>
            Name
          </th>
          <th>
            Email
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            John
          </td>
          <td>
            7XQp0@example.com
          </td>
        </tr>
        <tr>
          <td>
            Christ
          </td>
          <td>
            abcd@example.com
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
